lightspeed/modules/con_telemetry-data-collection-notice.adoc

```diff
@@ -0,0 +1,39 @@
+:_content-type: CONCEPT
+
+[id="telemetry-data-collection-notice_{context}"]
+= {LightspeedShortName} telemetry data collection notice
+
+In connection with your use of this Red Hat offering, Red Hat may collect telemetry data about your use of the software. This data allows Red Hat to monitor the software and to improve Red Hat offerings and support, including identifying, troubleshooting, and responding to issues that impact users. The data may also be used to enable you to track your entitlements to Red Hat subscriptions and take advantage of future Red Hat purchasing programs. It may also allow Red Hat to assist you in implementing upgrades to minimize service impact. The data may be shared internally within Red Hat to improve the user experience.  If you are evaluating Red Hat software, the data will help Red Hat determine if you need assistance.
+
+== What information does Red Hat collect?
+
+Tools within the software monitor various metrics and this information is transmitted to Red Hat.  The following metrics are monitored:
+
+* Organization you are logged into (Organization ID, account number)
+* Large language model (or models) that you are connected to 
+* Prompts and content suggestions, including accept or reject of the content suggestions
+* User sentiment feedback
+
+== Personal Data
+Red Hat does not intend to collect personal information. If Red Hat discovers that personal information has been inadvertently received, Red Hat will delete such information. To the extent that any telemetry data constitutes personal data, please refer to the link:https://www.redhat.com/en/about/privacy-policy[Red Hat Privacy Statement] for more information about Red Hat's privacy practices. 
```

Comment:
Again just a suggestion feel free to ignore. But another please here, I often find "you can" subs in nicely in most cases.

Reply:
This is done, thank you.